### DISCUSSION TITLE
New Jobs not starting

### DISCUSSION THREAD

**Discussion by ChopaChupChup** [swar/Swar-Chia-Plot-Manager#543] [Q&A]
System: Win10
The manager kicked two jobs with a stagger of 30 minutes out.
Everthing was correct, but now no new jobs will start.

The config-file didn't change and worked yesterday.
No I added a new CPU + Mainboard and it fails.

Restart of manager didn't help. Command-line runs under Administrator.

**Comment by leonidcan** [comment]
I have same issue, but I changed configuration (added new jobs and set max_plots: 0 to old job as per manual)
Now I only see original plots started during first session, please help!

**Comment by Crimsonking314** [comment]
The same thing happened to me and I got it to run again by issuing a restart command.  python manager.py restart

**Comment by talon33176** [comment]
Same issue but I was able to see further up the config file that we are limited to 14-15 concurrent plots and then it crashes.  Someone needs to get a hold of the author and let them know that they shouldn't be conservative with their concurrent plots.  It's why it's crashing.

**Reply by talon33176** [reply]
This is the problem right here


global:
      
      
          # These are the settings that will be used globally by the plot manager.
      
      
          #
      
      
          # max_concurrent: The maximum number of plots that your system can run. The manager will not kick off more than this
      
      
          #                 number of plots total over time.
      
      
          max_concurrent: 10
